apps/example-tailwind/src/app/pages/page-profile/page-profile.component.html

```diff
@@ -1,25 +1,712 @@
-<div class="p-6">
-    <div class="grid grid-cols-12 gap-4">
-        <div class="col-span-2">
-            <div class="px-1">
-                @for (item of userRoute; track item.id) {
-                    <ul class="space-y-1">
-                        <li class="mb-1 text-gray-500">
-                            <a
-                                class="hover:bg-primary-40 hover:text-primary-500 inline-block w-full rounded-sm px-3 py-1.5 transition-all duration-300"
-                                routerLinkActive="bg-primary-40 text-primary-500"
-                                [routerLink]="item.path"
-                                >{{ item.title }}</a
-                            >
-                        </li>
-                    </ul>
-                }
+<div class="p-5">
+    <div class="flex gap-6 pb-10">
+        <div class="">
+            <div>
```

Comment:
please remove empty class and remove extra div
```suggestion
        <div>
```